packages/components/src/hooks/useAutoLoadedData.test.ts

```diff
@@ -1,9 +1,9 @@
 import { LoadedState } from '@terra-ui-packages/core-utils';
 import { act, renderHook } from '@testing-library/react';
-import { UseLoadedDataResult } from 'src/libs/ajax/loaded-data/useLoadedData';
-import { controlledPromise } from 'src/testing/test-utils';
```

Comment:
What about the `controlledPromise` in `src/testing/test-utils`?

Reply:
To try to limit the changes in this PR, I copied it into components' test-utils module.

For a follow up, I intent to move it to core-utils and have both the components package and the app use it from there.

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4810.